### ISSUE TITLE
can't render multiline notes

### ISSUE THREAD

**Issue by aisbergde** [softwaretechnik-berlin/dbml-renderer#1]
I get this error

>D:\Repos\GitHub\DataHandwerk\DataHandwerk-toolkit-mssql\docs\src\dbml>dbml-renderer -i dhw_fk.dbml -o dhw_fk.svg
Could not parse input at line 15. Expected "," or "]" but "'" found.

These are the first few lines in the dbml script, the script is valid in dbdiagram.io

```dbml
Table "[dbo].[__RefactorLog]"{
"OperationKey" uniqueidentifier [not null, pk]
}
Table "[dbo].[sysdiagrams]"{
"diagram_id" int [not null, pk, increment]
"definition" varbinary(max) [ null]
"name" sysname [not null]
"principal_id" int [not null]
"version" int [ null]
}
Table "[graph].[ProcedureInstance]"{
"$node_id_3FC7AA841B564294B72482DE078E7EB2" nvarchar(1000) [not null]
"graph_id_3091B0F8FDFD4CCBAA8FF240634B29BA" bigint [not null]
"Instance" varchar(500) [not null]
"Procedure_fullname" nvarchar(261) [not null, Note: '''
(concat('[',[RepoObject_schema_name],'].[',[RepoObject_name],']'))
''']
"Procedure_RepoObject_guid" uniqueidentifier [not null]
}
Table "[graph].[ProcedureInstance_S]"{
"Procedure_RepoObject_guid" uniqueidentifier [not null, pk]
"Instance" varchar(500) [not null, pk]
"Procedure_fullname" nvarchar(261) [not null, Note: '''
(concat('[',[RepoObject_schema_name],'].[',[RepoObject_name],']'))
''']
}
Table "[graph].[ReferencedObject]"{
"$edge_id_25B0DD35E10A49DE8B49228E6802F3EB" nvarchar(1000) [not null]
"$from_id_7B1BEB60463741CAA681054A489A2714" nvarchar(1000) [ null]
"$to_id_26820C41BE034B00BCBC470518101233" nvarchar(1000) [ null]
"from_id_7F3CEF660D794A2A81530928BC4B36F1" bigint [not null]
"from_obj_id_A00C6316716D4F9DABE96A189F3A214A" int [not null]
"graph_id_85ECDD019A4C46AF80CFD8DDFA08F479" bigint [not null]
"to_id_45A2D6271B224A08B66CA0CF6A200A54" bigint [not null]
"to_obj_id_F1F1771130674EE9A9DAE06330740E13" int [not null]
}
```

Here is also the whole script as a test case:

[dhw_fk.zip](https://github.com/softwaretechnik-berlin/dbml-renderer/files/6051681/dhw_fk.zip)

**Comment by lucastorri** [comment]
Thank you, @aisbergde! I'm planning to go over the open tickets this weekend. Would you mind if I add your example to our test set in https://github.com/softwaretechnik-berlin/dbml-renderer/tree/master/examples?

Thank you!

**Comment by aisbergde** [comment]
@lucastorri of course you can add this script for testing. This is generated from an open source project and there is nothing to hide. And I will be happy to test your solution also with other databases. My idea is to create this kind of scripts for all my databases in the future. In the next step I will add some more virtual FK.

I also want to try to use dbml for visualization of referenced between entities or between columns to show data lineage.

And if this works then I will try to generate and visualize the dbml scripts for selected objects on the fly.

**Comment by lucastorri** [comment]
Thanks, @aisbergde!

 For now we added a simplified example that contains the issue, but later on we'll probably want to use the file as a big schema to test against.

We just released version [1.0.19](https://www.npmjs.com/package/@softwaretechnik/dbml-renderer/v/1.0.19) that contains fixes that can parse and generate SVGs with your example.

I'll close this issue for now, but please let us know if you encounter any other issue.
